src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java

```diff
@@ -64,6 +65,8 @@ private String getResourceDir(){
 
         return new Object[][]{
                 {new BQSRTest(hg18Reference, HiSeqBam, dbSNPb37, "", getResourceDir() + "expected.NA12878.chr17_69k_70k.txt")},
+                // TODO: add test once I find the reference
+                //{new BQSRTest(GRCh37Ref, trickyBam, dbSNPb37, "", getResourceDir() + "expected.CEUTrio.HiSeq.WGS.b37.ch20.4379150-4379157.recal.txt")},
```

Comment:
is this reference  `public static final String GRCH37_REF_ID = "EOSsjdnTicvzwAE";` ?